.github/workflows/ci.yml

```
@@ -213,9 +213,11 @@ jobs:
       - name: Install Intel® oneAPI DPC++/C++ Compiler
         if: matrix.cxx_compiler == 'dpcpp' || matrix.cxx_compiler == 'icx'
         shell: cmd
-        run: |
+        ### powershell cmnd in the following run termal is to set the tbb path in cl.cfg file" #####
```

Let's describe the purpose instead of the actions:
```suggestion
        # cl.cfg is modified to specify location of the library according to 
        # https://www.intel.com/content/www/us/en/developer/articles/release-notes/intel-oneapi-dpc-c-compiler-release-notes-2022.html
```

---

This can be  done in the PR description instead. When merging, please, leave a PR number for the reference in the commit message.